docs/publishing/sections/4_analytics_portfolio_site.md

```diff
@@ -9,3 +9,118 @@ For these situations, the [Jupyter Book-based](https://jupyterbook.org/en/stable
+    * This `.yml` file will include the directory to the notebook(s) you want to publish.
+    * Name your `.yml` file
+    * The structure of your `.yml` file depends on the type of your analysis:
+        * If you have one parameterized notebook with **one parameter**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            notebook: ./my-analyses/my-notebook.ipynb
+            parts:
+               - caption: Introduction
+               - chapters:
+                 - params:
+                      district_parameter: 1
+                      district_title: District 1
+            ```
+         * If you have a parameterized notebook with **multiple parameters**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            notebook: ./my-analyses/my-notebook.ipynb
+            parts:
+            - chapters:
+              - caption: County Name
+                params:
+                  paramter1_county_name
+                sections:
+                - city: parameter2_city_name
+                - city: parameter2_city_name
+            ```
+         * If you have an individual notebook with **no parameters**:
+
+            ```
+            title: My Analyses
+            directory: ./my-analyses/
+            readme: ./my-analyses/README.md
+            parts:
+            - caption: Introduction
+            - chapters:
+              - notebook: ./my-analyses/notebook_1.ipynb
+              - notebook: ./my-analyses/notebook_2.ipynb
+            ```
+## Building and Deploying your Report:
+### Build your Report:
+**Note:** The build command must be run from the root of the repo!
+1. Navigate back to the repo data-analyses and install the portfolio requirements with
+`pip install -r portfolio/requirements.txt`
+2. Then run `python portfolio/portfolio.py build my_report` to build your report
+    * The `my_report` will be replaced by the name of your `.yml` file in [data-analyses/portfolio/sites](https://github.com/cal-itp/data-analyses/tree/main/portfolio/sites).
+    * Your build will be located in: `data-analyses/portfolio/my_report/_build/html/index.html`
+4. Add the files using `git add` and commit your progress!
+
+
+### Deploy your Report:
+
+1. Make sure you are in the root of the data-analyses repo: `~/data-analyses`
+2. Run `python portfolio/portfolio.py build my_report --deploy`
+    * The `my_report` will be replaced by the name of your `.yml` file in [data-analyses/portfolio/sites](https://github.com/cal-itp/data-analyses/tree/main/portfolio/sites).
+3. Once this runs, you can check the preview link at the bottom of the output. It should look something like:
+    * `–no-deploy`: `file:///home/jovyan/data-analyses/portfolio/my-analysis/_build/html/index.html`
```

Comment:
I believe it's `my_report` instead of `my-analysis` here (underscore, not hyphen):
* `–no-deploy`: `file:///home/jovyan/data-analyses/portfolio/my_report/_build/html/index.html`